contrib/lowdown/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "lowdown"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "PREFIX=/usr",
+    "MANDIR=/usr/share/man",
+]
+make_check_target = "regress"
+pkgdesc = "Markdown translator"
+maintainer = "ttyyls <contact@behri.org>"
+license = "ISC"
+url = "https://kristaps.bsd.lv/lowdown"
+source = f"{url}/snapshots/{pkgname}.tar.gz"
+sha256 = "f31e3950c4732b1e409174fa092eca40c55be77a448ee2818df987979d7b0879"
+hardening = ["vis", "cfi"]
+
+
+def init_configure(self):
+    ldfl = self.get_ldflags(shell=True)
+    self.env["LDLFAGS"] = ldfl
```

Comment:
```suggestion
    configure_args += f"LDFLAGS={self.get_ldflags(shell=True)}"
```

they are in env already, the configure ignores the env

Reply:
`cannot access local variable 'configure_args' where it is not associated with a value`

Reply:
needed a self reference and I substituted the whole thing instead of mutating it. seems to work now.

Reply:
yeah i forgot the self. , that was the only change needed